### ISSUE TITLE
Base URL not Working

### ISSUE THREAD

**Issue by ShaheenDevelopers** [ibnulhayat/MuslimPro#1]
Aslam o Alaikum, hope you are doing well, your app doesn't work it's stuck on Splash Screen.

**Comment by ibnulhayat** [comment]
Yes bro, there are some problems in this application. Application isn't
complete fully.

**Comment by ShaheenDevelopers** [comment]
I also want to contribute if you interested because I have some modules but you're app not working yet.

**Comment by ibnulhayat** [comment]
Thank you. But it is not possible because this is a group project.

**Comment by ShaheenDevelopers** [comment]
Oh! Alright, No Problem Good Luck :-)
